standard/classes.md

```diff
@@ -490,7 +491,49 @@ Any constraints shall be consistent among dependent type parameters. If type par
 
 It is valid for `S` to have the value type constraint and `T` to have the reference type constraint. Effectively this limits `T` to the types `System.Object`, `System.ValueType`, `System.Enum`, and any interface type.
 
-If the `where` clause for a type parameter includes a constructor constraint (which has the form `new()`), it is possible to use the `new` operator to create instances of the type ([§12.8.17.2](expressions.md#128172-object-creation-expressions)). Any type argument used for a type parameter with a constructor constraint shall be a value type, a non-abstract class having a public parameterless constructor, or a type parameter having the value type constraint or constructor constraint.
+The token `notnull` is neither a keyword nor a contextual keyword. When it is encountered, it will either:
+
+- Bind to a type named `notnull`
```

Comment:
@gafter pointed out it could be an alias - we can just strike "named `notnull`", or see @Nigel-Ecma's wording in #1246.